docs/content/bundle/create.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Create a Bundle
+description: Create a bundle with Porter
+---
+
+Let's walk through how to create and customize your very own Porter bundle.
+A bundle includes both the tools and the scripts or logic necessary to automate the deployment.
+When writing a bundle, it is best if you have already figured out the commands necessary to perform the deployment first, and then use Porter to package that into a bundle.
+Learning Porter while also figuring out how to deploy a particular application can be difficult.
+
+## Requirements
+You must [install Porter], and optionally can use the [Porter Visual Studio Code] extension for autocomplete while editing the porter.yaml file.
+
+* [Create a Bundle](#create-a-bundle)
+* [Verify the Bundle](#verify-the-bundle)
+* [Install Tools](#install-tools)
+    * [Use Mixins](#use-mixins)
+    * [Use a Custom Dockerfile](#use-a-custom-dockerfile)
+* [Customize the Install Action](#customize-the-install-action)
+* [Test Your Bundle](#test-your-bundle)
+* [Publish Your Bundle](#publish-your-bundle)
+* [Use the Published Bundle](#use-the-published-bundle)
+
+## Create a Bundle
+
+Use the [porter create](/cli/porter_create) command to scaffold a new bundle in the current directory.
+The directory containing the files for the bundle is called the **bundle directory**.
+The generated bundle is very similar to the [hello example bundle] and prints out "Hello World" when installed.
+It does not allocate any resources and is safe to run and uninstall when you are finished.
+
+## Verify the Bundle
+
+Your bundle is ready to build and run!
+Let's do a quick check before making any further changes to verify that everything is working.
+
+1. Use the [porter build] command to build the bundle. 
+   This prepares the bundle so that it can be distributed over a registry by building the bundle image and packaging it in a bundle.
+2. Use the [porter install] command to run the bundle's install action defined in the porter.yaml file.
+
+   ```console
+   $ porter install mybundle
+   executing install action from porter-hello (installation: /mybundle)
+   Install Hello World
+   Hello World
+   execution completed successfully!
+   ```
+
+   You do not need to specify the \--reference flag with the install command because the current directory is a bundle directory, containing a porter.yaml file.
```

Comment:
Why does this `--reference` isn't quoted with code syntax while we have the one being so in line 150?